packages/ra-ui-materialui/src/detail/TabbedShowLayout.stories.tsx

```diff
@@ -22,47 +26,73 @@ const record = {
     year: 1869,
 };
 
-export const Basic = () => (
-    <TestMemoryRouter>
-        <ResourceContext.Provider value="books">
-            <RecordContextProvider value={record}>
-                <TabbedShowLayout>
-                    <TabbedShowLayout.Tab label="First">
-                        <TextField source="id" />
-                        <TextField source="title" />
-                    </TabbedShowLayout.Tab>
-                    <TabbedShowLayout.Tab label="Second">
-                        <TextField source="author" />
-                        <TextField source="summary" />
-                        <NumberField source="year" />
-                    </TabbedShowLayout.Tab>
-                </TabbedShowLayout>
-            </RecordContextProvider>
-        </ResourceContext.Provider>
+const Wrapper = ({
+    children,
+    record = data,
+}: {
+    children: React.ReactNode;
+    record?: RaRecord;
+}) => (
+    <TestMemoryRouter
+        initialEntries={[`/books/${encodeURIComponent(record.id)}/show`]}
+    >
+        <AdminContext
+            i18nProvider={{
+                translate: (x, options) => options?._ ?? x,
+                changeLocale: () => Promise.resolve(),
+                getLocale: () => 'en',
+            }}
+            dataProvider={testDataProvider({
+                // @ts-ignore
+                getOne: () => Promise.resolve({ data: record }),
+            })}
+            defaultTheme="light"
+        >
+            <ResourceContextProvider value="books">
+                <Routes>
+                    <Route
+                        path="/books/:id/show/*"
+                        element={<Show sx={{ width: 600 }}>{children}</Show>}
+                    />
+                </Routes>
+            </ResourceContextProvider>
+        </AdminContext>
     </TestMemoryRouter>
 );
 
+export const Basic = () => (
+    <Wrapper>
```

Comment:
I had to add a Wrapper to all stories to render the tabs in a nested route, otherwise the new implementation does not work.